index.js

```diff
@@ -23,6 +23,9 @@ var func = require('rework-plugin-function');
 module.exports = function(fn) {
   return func({
     url: function(path){
+      if (path.indexOf('data:image') == 0) {
```

Comment:
The return value here will always be a number so ===  should be used (as it is generally preferred over == equal)

Reply:
@techwes resolved that